scripts/test-local.sh

```diff
@@ -1,5 +1,11 @@
 #!/usr/bin/env bash
 
+glob=$*
+if [ "$glob" == "" ]; then
+  glob="test/local test/remote"
+fi
+
 ./scripts/gen_keys.js
 ./scripts/check-i18n.js
-./scripts/tap-coverage.js test/local test/remote 2>/dev/null
+./scripts/tap-coverage.js $glob 2>/dev/null
+grunt
```

Comment:
According to the [docs](https://docs.npmjs.com/misc/scripts#path), that directory should be added to the `PATH`, which should affect `sh`'s lookup as well.